TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriveTrain.java

```diff
@@ -0,0 +1,651 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.kauailabs.navx.ftc.AHRS;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.Range;
```

Comment:
#fix change to "import com.qualcomm.robotcore.*"

Reply:
I tried updating the code to this, it returns an error and tries to autofill with these same imports again.